src/main/java/com/codetaylor/mc/onslaught/modules/onslaught/ModuleOnslaught.java

```diff
@@ -6,24 +6,80 @@
+import com.codetaylor.mc.onslaught.modules.onslaught.invasion.InvasionCommandStarter;
+import com.codetaylor.mc.onslaught.modules.onslaught.invasion.InvasionCompletionPercentageCalculator;
+import com.codetaylor.mc.onslaught.modules.onslaught.invasion.InvasionCounter;
+import com.codetaylor.mc.onslaught.modules.onslaught.invasion.InvasionFinishedPredicate;
+import com.codetaylor.mc.onslaught.modules.onslaught.invasion.InvasionKillCountUpdater;
+import com.codetaylor.mc.onslaught.modules.onslaught.invasion.InvasionMessageSender;
+import com.codetaylor.mc.onslaught.modules.onslaught.invasion.InvasionMessageSenderWarning;
+import com.codetaylor.mc.onslaught.modules.onslaught.invasion.InvasionPlayerData;
+import com.codetaylor.mc.onslaught.modules.onslaught.invasion.InvasionPlayerDataFactory;
+import com.codetaylor.mc.onslaught.modules.onslaught.invasion.InvasionPlayerTimerValueSupplier;
+import com.codetaylor.mc.onslaught.modules.onslaught.invasion.InvasionSpawnDataConverterFunction;
+import com.codetaylor.mc.onslaught.modules.onslaught.invasion.InvasionStopExecutor;
+import com.codetaylor.mc.onslaught.modules.onslaught.invasion.InvasionTimestampFunction;
+import com.codetaylor.mc.onslaught.modules.onslaught.invasion.InvasionWarningMessageTimestampFunction;
 import com.codetaylor.mc.onslaught.modules.onslaught.invasion.render.InvasionHudRenderInfo;
 import com.codetaylor.mc.onslaught.modules.onslaught.invasion.render.client.InvasionHudRenderer;
 import com.codetaylor.mc.onslaught.modules.onslaught.invasion.sampler.SpawnSampler;
 import com.codetaylor.mc.onslaught.modules.onslaught.invasion.sampler.predicate.SpawnPredicateFactory;
 import com.codetaylor.mc.onslaught.modules.onslaught.invasion.selector.InvasionSelectorFunction;
-import com.codetaylor.mc.onslaught.modules.onslaught.invasion.spawner.*;
+import com.codetaylor.mc.onslaught.modules.onslaught.invasion.spawner.ActiveMobCounter;
+import com.codetaylor.mc.onslaught.modules.onslaught.invasion.spawner.ActiveWavePredicate;
+import com.codetaylor.mc.onslaught.modules.onslaught.invasion.spawner.DeferredSpawnClientParticlePacketSender;
+import com.codetaylor.mc.onslaught.modules.onslaught.invasion.spawner.DeferredSpawnData;
+import com.codetaylor.mc.onslaught.modules.onslaught.invasion.spawner.DeferredSpawnEffectApplicator;
+import com.codetaylor.mc.onslaught.modules.onslaught.invasion.spawner.DeferredSpawner;
+import com.codetaylor.mc.onslaught.modules.onslaught.invasion.spawner.EntityInvasionDataInjector;
+import com.codetaylor.mc.onslaught.modules.onslaught.invasion.spawner.Spawner;
+import com.codetaylor.mc.onslaught.modules.onslaught.invasion.spawner.SpawnerMob;
+import com.codetaylor.mc.onslaught.modules.onslaught.invasion.spawner.SpawnerMobForced;
+import com.codetaylor.mc.onslaught.modules.onslaught.invasion.spawner.SpawnerWave;
+import com.codetaylor.mc.onslaught.modules.onslaught.invasion.spawner.WaveDelayTimer;
```

Comment:
Indeed it is. But there's a strong argument for this:

If you have 2 packages that import `*`, if they happen to have 2 imports with overlapping names, it won't compile. More sinister- it may at that point in time, but one of those 2 packages introduces a duplicate later. Now suddenly the project stops compiling.

The Google Style Guide explicitly excludes wildcard imports for these reasons. Sonar, a code quality analyzer treats it by default as a [critical code smell RSPEC-2208](https://rules.sonarsource.com/java/RSPEC-2208).